dask/base.py

```diff
@@ -932,13 +932,17 @@ def normalize_series(s):
         return [
             s.name,
             s.dtype,
-            normalize_token(s._data.blocks[0].values),
+            normalize_token(s._values),
             normalize_token(s.index),
         ]
 
     @normalize_token.register(pd.DataFrame)
     def normalize_dataframe(df):
-        data = [block.values for block in df._data.blocks]
+        mgr = df._data
+        if hasattr(mgr, "arrays"):
+            data = list(mgr.arrays)
```

Comment:
If possible, I'd prefer a version check here rather than a hasattr, so that we can clean this up in the future. Will this be pandas 1.3.0 and newer?

I think we have a `dask.dataframe._compat` where we define constants for the pandas version. Might need `1.3.0.dev0`.

Reply:
A version check is fine as well (it would indeed be 1.3.0 and newer). But, that would then rely on the fact that a BlockManager has a `.arrays` attribute (which I for now mainly added for testing purposes / to be able to share some code), and I am not fully sure if downstream packages should rely on that.

Reply:
Updated to use an explicit version check. 

I didn't use `dask.dataframe._compat` since this base.py is outside the `dask.dataframe` module.